src/App.css

```diff
@@ -0,0 +1,39 @@
+.App {
```

Comment:
- [ ] When I try to open the mobile menu from the "Add motorcycle" page, instead of the nav covering the screen, this happens:

![image](https://github.com/jonas-45/motorcycle-booking-react/assets/57421823/d8cada0f-99e5-45e2-a803-a6e35fd4ade7)

Kindly, make it so the navbar on this page shows like on the other pages, to improve consistency and the UX . 💪